roles/kuryr/templates/configmap.yaml.j2

```diff
@@ -587,6 +587,18 @@ data:
     # Time (in seconds) the watcher retries watching for. (integer value)
     watch_retry_timeout = 3600
 
+
+{% if (kuryr_controller_replicas|default(1)|int > 1) %}
```

Comment:
This is in the wrong location. It should be on kuryr.conf instead of kuryr-cni.conf section

Reply:
Fixed!